owasp-dependency-check-suppressions.xml

```diff
@@ -359,6 +359,9 @@
     ]]></notes>
     <cve>CVE-2022-45855</cve>
     <cve>CVE-2022-42009</cve>
+    <!-- Suppress hadoop CVEs that not applicable to hadoop-annotations -->
```

Comment:
we should add a comment as to why these are not applicable to hadoop-annotations.

Reply:
Updated it! Thanks, @abhishekagarwal87